infogami/infobase/client.py

```diff
@@ -205,6 +205,10 @@ def _get(self):
 
     def __getattr__(self, key):
         return getattr(self._get(), key)
+    
+    def __iter__(self):
```

Comment:
I recommend delegating to `x.__iter__()` instead, i.e.,

`return self._get().__iter__()`

The approach here works OK, but creates an unnecessary layer IMHO.

Not a showstopper.

Reply:
Nice!  Done.